### PR TITLE
Qa capstone PR

### PR DESCRIPTION
**Pull request by gdvl777**
Project requirements
You should personalize the content of your page. Choose a topic that is different than the one in the original design.

You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

Colors.
Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
Layout: composition and space between elements.
The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

You can use a CSS framework (for example Bootstrap) for styling, if you want to.

You should build only these 2 pages:

The home page.
The about page.
Each of these pages should have versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.
Interactions

Links
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.

Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.
Dynamic page

The section "Featured speakers" should be created dynamically in JavaScript.
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @gdvl777 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request with most linters tests failing. Kindly ensure that all the linters shows green (passed) before submitting for another review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+New for 2023, experience an unparalleled atmosphere and an unbeatable view on all the action – THE SUITES at SINGAPORE SMASH – your bespoke, high-end table tennis experience.<br>
+
+Enjoy first-class entertainment and adrenaline-filled performances with staff or clients, all from an immersive fully serviced corporate box located above the player walkout runway.
+        </p>
+        <p class="date">2023.10.13(WED) ~ 15(FRI)</p>
+        <p class="place">@Singapore, Singapore.</p>
+      </div>
+    </div>
+
```
Comment (Owusu-Desmond) [inline index.html:102]: - [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, your nav padding, your background design, text style, and alignment of elements do not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your Design on desktop
image

### Cindy Design on desktop
image

### Your Design on mobile
image

### Cindy's design on mobile
image

**Review by gdvl777 — COMMENTED** [review]
Everything ok! Good job so far

**Comment by gdvl777** [comment]
Everything ok

File: index.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_03.png" alt="Forum Icon"></td>
+              <td class="t-title">Special Perks</td>
+              <td class="t-description">Backstage tour, training room, memorabilia and more.</td>
+            </tr>
+          </tbody>
+        </table>
+
```
Comment (Owusu-Desmond) [inline index.html:138]: - [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, your `SUITES` section suppose to have 5 categories of containers, your images are not showing, you are missing the `SEE THE WHOLE PROGRAM` text and you have no background design which makes your design not matching [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your design
image

### Cindy's design
Screenshot from 2023-03-25 23-58-15

File: index.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    </section>
+      
+
+    <section class="speakers" id="speakers">
+      <div class="featured">
+        <h4>Top Players</h4>
+      </div>
+      <div class="main-bar"></div>
+        
+      <div class="grid-container">
```
Comment (Owusu-Desmond) [inline index.html:171]: - [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, in your top players section your images are not showing and your margin and text style does not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your design
image


### Cindy's design
Screenshot from 2023-03-26 00-07-44

File: about.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+  <main class="main">
+    <section class="intro-about">
+      <div class="intro">
+        <h2 class="greet-about">EXPERIENCE THE SINGAPORE SMASH IN STYLE</h2>
+        <h1 class="title-about">Singapore Smash 2023<br>World Table Tennis 2023</h1>
+        <div class="bar-about-main"></div>
+        <div class="about">
+          <p class="description-about">
+            Men's and Women's Singles will feature 64 players each in the Main Draw. 50 players will be automatically entered based on ITTF World Table Tennis Ranking. <br>
```
Comment (Owusu-Desmond) [inline about.html:98]: - [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, your nav padding, your background design, text style, and alignment of elements do not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 
  
  ### Your Design
  image
  
  ### Cindy's design
  image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+      <div class="conference-images">
+        <div class="conference1">
+          <h6 class="conference-year">2022</h6>
+          <p class="conference-prev">SMASH WTT in Dubai</p>
+        </div>
+        <div class="conference2">
+          <h6 class="conference-year">2021</h6>
+          <p class="conference-prev">SMASH WTT  in Lisboa</p>
+        </div>
+
```
Comment (Owusu-Desmond) [inline about.html:158]: - [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, the blurred images, logo are not showing which makes your project design not matching [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your Design
image


### Cindy's design
image

File: README.md [hunk 6]
```diff
@@ -55,179 +30,141 @@ After you're finished please remove all the comments and instructions!
...
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- Here you will see a [Live Demo](https://oovillagran.github.io/Capstone_M1/) of my project.
+
+- This is the [Loom Walktrough Video](https://www.loom.com/share/1f689459a7134b71846c7d1c1bd3d778) of my project.
```
Comment (Owusu-Desmond) [inline README.md:88]: - [ ] Please this recorded video is not a video of your project so make sure that you record a video of this project as it is required in the project requirement. :warning:

File: README.md [hunk 7]
```diff
@@ -55,179 +30,141 @@ After you're finished please remove all the comments and instructions!
...
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- Here you will see a [Live Demo](https://oovillagran.github.io/Capstone_M1/) of my project.
+
```
Comment (Owusu-Desmond) [inline README.md:87]: - [ ] Hey this live demo link is not your project so make sure that you replace it with your project live demo link :warning:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @gdvl777 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**